relayer/package.json

```diff
@@ -18,11 +18,12 @@
     "lint:fix": "eslint --ext .js,.ts --ignore-path .gitignore --fix .",
     "test:unit": "jest unit",
     "test:coverage": "jest --collect-coverage",
-    "generate-types": "typechain --target=ethers-v5 ./src/assets/abi/**.json --out-dir=./types-gen/contracts/",
+    "generate-types": "typechain --target=ethers-v5 ./node_modules/@beamer-bridge/deployments/dist/abis/mainnet/**.json --out-dir=./types-gen/contracts/",
```

Comment:
I am a little bit worried about this as well. This also means that we will not be able to integrate the relayer in our local e2e tests with the local ABIs.

I am trying to think of ways to avoid this. We could have a `prepare` or `configure` script for generating the contract types / ABIs based on a provided `--path-to-deployments` argument. So, before running the relayer, one would have to run this script (yarn process?) which will generate the types and dump them in the current `types-gen/contracts` folder.

Same goes for the build process.

Reply:
The reason why i am proposing such a solution with an argument `--path-to-deployments` is because the e2e tests are storing the deployment artifacts outside of the project directory (temporary folders) which makes it trickier to automate this process by using only the `@beamer-bridge/deployments` package ..

Reply:
Tested this out, but unfortunately it only extends the glob patterns instead of overwriting the argument value.